docs/LwM2M_Client/OpenThread/OTBR_with_svetovid.md

```diff
@@ -0,0 +1,183 @@
+
+    === "**nRF52840-dongle**"
+        Install `nrf5sdk-tools`
+        ```
+            nrfutil install nrf5sdk-tools
+        ```
+
+        Generate the RCP firmware package
+        ```
+            nrfutil pkg generate --hw-version 52 --sd-req=0x00 --application build/bin/ot-rcp.hex --application-version 1 build/bin/ot-rcp.zip
+        ```
+
+        Press the `reset` button to enter the bootloader mode, the onboard LED should start pulsing red.
+
+        Check the dongle path
+        ```
+            ls -l /dev/ttyACM*
+        ```
+
+        Flash the dongle (assuming that the dongle path is `/dev/ttyACM0`)
+        ```
+            nrfutil dfu usb-serial -pkg build/bin/ot-rcp.zip -p /dev/ttyACM0
+        ```
+
+## Connecting to the LwM2M Server
+
+To connect to {{ coiote_long_name }}, please register at [{{ coiote_site_link }}]({{ coiote_site_link }}).
+
+To connect the device:
+
+1. Log in to {{ coiote_short_name }} and from the left side menu, select **Device Inventory**.
+2. In **Device Inventory**, click **Add device**.
+3. Select the **Connect your LwM2M device directly via the Management server** tile.
+       ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+    3. In the **Device credentials** step:
+         - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
+             ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
+              - In the **Key identity** field, type the same name as in the `Endpoint name` field.
+              - In the **Key** field, type the shared secret used in the device-server authentication.
+    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+    5. In the **Connect your device** step, follow the next sections to run the client and connect it to the server.
+
+## Switch to the target device
+
+If you haven't worked on your target device thus far, it's time to switch to it, in our case it is the `Raspberry Pi 3B`. Connect the RCP to the target device and proceed with this tutorial while working on it.
+
+## Downloading Docker image
+
+Download the newest version of `svetovid-with-otbr` docker image
+```
+    docker pull avsystem/svetovid-with-otbr
+```
+
+## Environmental variables
+
+To make it easier to run the docker container, we will set a bunch of environmental variables.
+
+At first let's check the RCP path
+```
+    ls -l /dev/ttyACM*
+```
+
+And set `RCP_PATH` variable, for example `/dev/ttyACM0`
+```
+    RCP_PATH='/dev/ttyACM0'
+```
+
+Now we will set variables related to LwM2M server and credentials:
+
+0. Server URI (e.g. {{ coiote_server }})
+```
+    SERVER_URI='{{ coiote_server }}'
+```
+
+0. Endpoint name (it will also be used as an DTLS Identity)
```

Comment:
A simple note that those names should be identical as in the Coiote will be helpful I think.